lib/src/components/list_item/notification_list_item.dart

```diff
@@ -148,8 +148,7 @@ class ZetaNotificationListItem extends ZetaStatelessWidget {
                               crossAxisAlignment: CrossAxisAlignment.start,
                               children: [
                                 Row(
-                                  mainAxisAlignment:
-                                      MainAxisAlignment.spaceBetween,
+                                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
```

Comment:
why is all this formatting happening?

Reply:
the last file to touch this file was from the fork and so the lint format changes didnt happen